{
    "has_execution_error": true,
    "execution_output": "Traceback (most recent call last):\n  File \"/home/rma336/calendar-planning/source/generated_code_calendar_2025-06-19 03:30:20.py\", line 78, in <module>\n    main()\n  File \"/home/rma336/calendar-planning/source/generated_code_calendar_2025-06-19 03:30:20.py\", line 68, in main\n    proposed_time = find_meeting_time(start_time, end_time, duration, schedules)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/rma336/calendar-planning/source/generated_code_calendar_2025-06-19 03:30:20.py\", line 35, in find_meeting_time\n    if (block_start <= time and time <= block_end) or (block_start <= time + duration and time + duration <= block_end):\n        ^^^^^^^^^^^^^^^^^^^\nTypeError: '<=' not supported between instances of 'int' and 'datetime.datetime'",
    "pred": null,
    "gold": {
        "day": "Monday",
        "time_range": "{11:30:12:00}"
    },
    "status": "Error",
    "violated_constraint": {
        "missing_fields": true
    },
    "is_exact_match": false,
    "constraints_satisfied": false,
    "pass_number": 4,
    "timing": {
        "api_call_time": 24.207669019699097,
        "execution_time": 0.01200246810913086,
        "constraint_eval_time": 1.430511474609375e-06,
        "pred_extract_time": 1.1920928955078125e-06,
        "gold_extract_time": 1.1682510375976562e-05,
        "token_count": 0
    }
}